workshop/faq/README.md

```diff
@@ -8,6 +8,8 @@
   - [Why do I see the following when I push the python app to IBM Cloud `**ERROR** Could not install python`](#why-do-i-see-the-following-when-i-push-the-python-app-to-ibm-cloud-error-could-not-install-python)
     - [Accessing Files](#accessing-files)
     - [Importing Files into the Cloud Pak for Data as a Service platform](#importing-files-into-the-cloud-pak-for-data-as-a-service-platform)
+    
```

Comment:
I believe this pre-work version of the workshop will go live at midnight, on the night before the workshop (Midnight Nov 23rd). Should you put the hours (and Time Zone) that you will be on the Webex to support the users? Also, you might want to put this at the very top of the FAQ, since that will make it easiest for the users to find.
Otherwise this seems like a good idea.